public/app/features/dashboard-scene/conditional-rendering/types.ts

```diff
@@ -11,6 +11,14 @@ import { ConditionalRenderingGroup } from './ConditionalRenderingGroup';
 import { ConditionalRenderingTimeRangeSize } from './ConditionalRenderingTimeRangeSize';
 import { ConditionalRenderingVariable } from './ConditionalRenderingVariable';
 
+export interface ConditionEvaluationResult {
+  // Actual result of the condition evaluation
+  result: boolean;
+
+  // When set, this flag will ignore the Show or Hide actions and the result is used as is
+  force: boolean;
```

Comment:
can't say I really understand this, what is ignored, and what is the scenario/problem it solves?